File: livingrimoire start here/LivinGrimoire java/src/Skills/special_skills/SkillBranch1Liner.java
package Skills.special_skills;

import LivinGrimoire.Skill;

public class SkillBranch1Liner extends SkillBranch {
    public SkillBranch1Liner(String goal, String defcon, int tolerance, Skill... skills) {
        super(tolerance);
        this.addGoal(goal);
        this.addDefcon(defcon);
        for (Skill skill : skills) {
            this.addSkill(skill);
        }
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/skills/special/SkillBranch1Liner.kt
package skills.special

import livinGrimoire.Skill

class SkillBranch1Liner(goal: String, defcon: String, tolerance: Int, vararg skills: Skill) :
    SkillBranch(tolerance) {
    init {
        addGoal(goal)
        addDefcon(defcon)
        for (skill in skills) {
            addSkill(skill)
        }
    }
}